crates/biome_resolver/src/lib.rs

```diff
@@ -581,7 +581,7 @@ fn resolve_path_info(
     }
 }
 
-fn is_relative_specifier(specifier: &str) -> bool {
+pub fn is_relative_specifier(specifier: &str) -> bool {
```

Comment:
This does not need to public?

Reply:
True, but I think I’ll leave it. It’s a useful utility anyway.